src/components/LoadingPage/LoadingPage.styles.ts

```diff
@@ -11,9 +13,7 @@ const rotate = keyframes`
   }
 `;
 
-export const Text = styled.h1`
-  font-family: 'Baloo 2', cursive;
-  font-weight: 700;
+export const LoadingText = styled(Text)`
```

Comment:
Like this is nicer than StyledLoadingText

Reply:
Sidenote: I dont really like these Blablabla.styles.ts -files.
I would place the components in files

Reply:
I know you did not write it ;)

Reply:
I agree! This took only a minute to fix! 😁